checks/raw/binary_artifact.go

```diff
@@ -278,7 +278,7 @@ func checkWorkflowValidatesGradleWrapper(path string, content []byte, args ...in
 	return true, nil
 }
 
-// fileExists checks if a file of name name exists, including within
+// fileExists checks if a file of name exists, including within
```

Comment:
Actually, this is probably better as:
```suggestion
// fileExists checks if a file named `name` exists, including within
```